src/features/iam/iamHelpers.ts

```diff
@@ -31,7 +32,38 @@ async function createDefaultProject(): Promise<void> {
   ext.explorerTree.refresh()
 }
 
+const renameProject = async ({ projectId }: { projectId: string }) => {
+  const project = await iamState.getProjectById(projectId)
+  if (!project) return
+  const name = await window.showInputBox({
```

Comment:
do we have some limitations for project name?

cc @vitalyrudenko-affinidi @carlos-affinidi

Reply:
Not at the moment

Reply:
is this ok for security purposes to be able to put anything?

Reply:
@OleksiiO-affinidi not that I know of... Let me have a look and get back to you. Should we implement some if there is none in place?

Reply:
this is not related directly to this MR
it is more like a general question
because if it is an issue it should be fixed on BE as well as on FE   
if it is not an issue then we should do nothing

Reply:
> is this ok for security purposes to be able to put anything?

As long as we escape any malicious content – I think so. React and VS Code do that by default.

Of course, we'll need to add some limitations in the future (for length, for some characters), but currently – no limits.